### DISCUSSION TITLE
Logs are not sent to loki without labels

### DISCUSSION THREAD

**Discussion by dbazhal** [vectordotdev/vector#19459] [Q&A] [answered]
### A note for the community

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Problem

Hey guys!
I found this some old threads on loki and labels and I still don't get the labels.
Modern loki sink asks labels as required field, and I suppose without them, vector just doesn't send logs to the loki right.
What should I fill into labels? Can I just put 
```
labels:
  "*": "*"
```
into a loki sink config to make it send all labels as they are?

### Configuration

```text
loki:
      type: loki
      endpoint: http://loki-gateway.loki.svc.cluster.local:80
      encoding:
        codec: json
      labels:
        "*": ""
      inputs:
        - container_json
```
```


### Version

0.32.1

### Debug Output

_No response_

### Example Data

_No response_

### Additional Context

_No response_

### References

_No response_

**Comment by jszwedko** [comment]
Hi @dbazhal ,

You need to configure what to send as the labels. For example:

```
labels:
  foo: "{{ bar }}"
```

You can use wildcards, but you still need to identify the keys/values you want to use as labels. See https://vector.dev/docs/reference/configuration/sinks/loki/#label-expansion for details.

Hope this helps!

**Reply by dbazhal** [reply, marked as answer]
Oh, thank you, sorry for disturbing, I figured it out, it turned out that I've been rendering label values empty because of using helm and my
```
labels:
  a: "{{ a }}
```
turned into "function does not exist"
and
```
labels:
  a: "{{ .a }}
```
turned into empty label value.
Made it with
```
labels:
  a: "{{ `{{ a }}` }}"
```
Now it all works.

**Reply by surekha3** [reply]
@dbazhal I got same issue with loki sink, solution which you state working, for me its just taking value as {{ a }} not actual value of a
can you please help how this worked for you
